scripts/clear-dnf.conf

```diff
@@ -0,0 +1,35 @@
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+[main]
+cachedir=/var/cache/yum/clear/
+keepcache=0
+debuglevel=2
+logfile=/var/log/yum.log
+exactarch=1
+obsoletes=1
+gpgcheck=0
+plugins=0
+installonly_limit=3
+reposdir=/root/mash
+
+[clear]
+name=Clear
+failovermethod=priority
+baseurl=https://download.clearlinux.org/current/x86_64/os/
+enabled=1
+gpgcheck=0
```

Comment:
Ideally gpg signature checking would be enabled for `[clear]` and `[main]`.

If we cannot enable this, we really need comments here and a big scary warning in the `README.md` explaining why it is not enabled, and the potential associated risks.